src/gl/pointFeature.js

```diff
@@ -279,34 +280,34 @@ geo.gl.pointFeature = function (arg) {
     }
 
     sourcePositions.pushBack(buffers.get("pos"));
-    geom.addSource(sourcePositions);
+    geom.addSource(sourcePositions, "pos");
```

Comment:
@manthey can we do this instead: 

sourcePositions = vgl.sourceDataP3fv("pos") 

Basically at the time of the creation of source array, we pass an argument. If not then internally we should create a name for it (using may be time of the creation)? I think this might be better.